### PR TITLE
Sectional contrast fix: Added bounding box around each traffic icon for greater visibility (e.g., on sectionals)

### PR DESCRIPTION
**Pull request by shanelenagh**
Added bounding box and numerous other color tweaks for greater sectional contrast. Also various other icon changes after conversations with Zubair.

### CODE REVIEW

**Comment by shanelenagh** [comment]
I am done tweaking this, @apps4av , so you can review and merge when you feel comfortable.

**Comment by shanelenagh** [comment]
Actually, @apps4av , hold off on this for a bit.  I am doing some tests on my phone that are showing a big penalty for the shadow effect--trying to see if it can be reduced, or if we have to go back to another strategy (e.g., the outline and transluscent box this PR started with).

**Comment by shanelenagh** [comment]
Ok, @apps4av , I am finally done messing with this (per our email discussions).  It is ready for your review.

File: lib/gdl90/traffic_cache.dart [hunk 1]
```diff
@@ -156,124 +156,179 @@ class TrafficCache {
...
+  /// configuration of enabled features.  This is used to determine UI-relevant state changes for repainting,
+  /// as well as the key to the picture cache  
+  String _iconStateKey = "";
 
-  _TrafficPainter(Traffic traffic) 
+  TrafficPainter(Traffic traffic) 
     : _aircraftType = _getAircraftIconType(traffic.message.emitter), 
       _isAirborne = traffic.message.airborne,
-      _flightLevelDiff = _getGrossFlightLevelDiff(traffic.message.altitude), 
+      _flightLevelDiff = prefAltDiffOpacityGraduation ? _getGrossFlightLevelDiff(traffic.message.altitude) : -999999,
```
Comment (apps4av) [inline lib/gdl90/traffic_cache.dart:275]: Use infinity instead?

**Review by apps4av — APPROVED** [review]

File: lib/gdl90/traffic_cache.dart (hunk 1 above)
Comment (shanelenagh) [reply lib/gdl90/traffic_cache.dart:275]: I really wish Dart had some of those constants that Java does (Integer.MAX_VALUE) for things like this, but it doesn't (https://stackoverflow.com/questions/32338554/is-there-an-int-infinity-for-dart).  So I picked a value that was well out of the range of what that function could return, due to its nature, and hopefully provides some indication it is a default value to populate a final field.  But maybe something that might be better served as an explanatory constant as well (say in the Constants.dart file).

**Review by apps4av — APPROVED** [review]

**Pull request merged**